models/spike_generator.h

```diff
@@ -212,7 +213,9 @@ class spike_generator : public DeviceNode
   port send_test_event( Node&, rport, synindex, bool );
   void get_status( DictionaryDatum& ) const;
   void set_status( const DictionaryDatum& );
+  void update_from_backend( std::vector< double > input_spikes ) override;
```

Comment:
Marking only this function `override` is inconsistent. If this is marked `override`, then the following functions should also be marked `override`:
- `has_proxies`
- `get_element_type`
- `send_test_event`
- `get_status`
- `set_status`
- `get_type`
- `event_hook`
- `sends_signal`
- `init_state_`
- `init_buffers_`
- `calibrate`
- `update`

Reply:
Do you prefer to indicate all the function override or not?

Reply:
Marking all functions listed `override` is preferred, because it better conveys the intent and can help catch bugs at compile time.

Reply:
done